packages/desktop-client/src/components/budget/BudgetCategories.jsx

```diff
@@ -315,7 +317,7 @@ export const BudgetCategories = memo(
               );
               break;
             default:
-              throw new Error('Unknown item type: ' + item.type);
+              throw new Error(t('Unknown item type: ') + item.type);
```

Comment:
Generally speaking, I think we've avoided translating error messages unless the user will actually see them in normal usage (so, "Failed to sync" is valid, "Developer error: programming mistake" is not). Do you know if this one can ever be shown?